tests/rptest/tests/cluster_health_overview_test.py

```diff
@@ -92,10 +97,15 @@ def one_node_down():
                 assert [self.redpanda.idx(first_down)] == hov['nodes_down']
                 # next check is "in" instead of "==" because we may also have under_replicated_partitions
                 assert 'nodes_down' in hov['unhealthy_reasons']
-                return True
-            return False
+                return True, hov
+            return False, None
+
+        hov = wait_until_result(one_node_down, 30, 2)
 
-        wait_until(one_node_down, 30, 2)
+        # when one node is down some partitions with replication factor of 1
+        # should be reported as leaderless
+        assert len(hov['leaderless_partitions']) > 0
+        assert hov['leaderless_count'] > 0
```

Comment:
Later on we were checking that more partitions go leaderless when the second node goes down. This check is now broken. We need to make sure RF-3 partitions actually go leaderless. Maybe create many more of them and check that the number of leaderless partitions is more than the number of RF-1 partitions?